app/components/article-comments.js

```diff
@@ -68,7 +68,7 @@ export default Component.extend(InViewportMixin, {
 
     this._super(...arguments);
 
-    if (page !== null) {
+    if (page !== null && page !== undefined) {
```

Comment:
In an old branch of mine, I added a basic test for this component: https://github.com/Wikia/mobile-wiki/blob/ember-octane-update/tests/integration/components/article-comment-test.js.

Do you think it could be extended to cover this case?

Reply:
This could be written as `page == undefined` as well I think, although I'm not 100% that it is better.

Reply:
> In an old branch of mine, I added a basic test for this component: https://github.com/Wikia/mobile-wiki/blob/ember-octane-update/tests/integration/components/article-comment-test.js.
> 
> Do you think it could be extended to cover this case?

I will add these tests here and extend for this case